dts/bindings/watchdog/nuvoton,npcx-watchdog.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-TWD node
+
+compatible: "nuvoton,npcx-watchdog"
+
+include: [base.yaml]
+
+properties:
+    reg:
+        required: true
+    label:
+        required: true
+    t0_out:
+        type: phandle
+        required: true
+        description: mapping item between wui and t0-out timer expired signal
```

Comment:
Please delete

Reply:
Done.

Reply:
>> Please delete
> Done.

Did you forget to push?

Reply:
Sorry. I misunderstand what you mean. (At the beginning, I thought you suggest me to delete the empty line next to line 18.) 
But I think what you suggest should be removing the reduplicative "description" property. I will push it now.

Reply:
Hope it is correct this time. And any further comment is appreciated.